src/initialization/InitDistribution.cpp

```diff
@@ -236,6 +236,9 @@ namespace impactx
         } else if (particle_type == "proton") {
             qe = 1.0;
             massE = ablastr::constant::SI::m_p / ablastr::constant::SI::MeV_invc2;
+        } else if (particle_type == "Hminus") {
+            qe = -1.0;
+            massE = 939.294308;  // value used in TraceWin
```

Comment:
We probably need to make this a bit more systematic in future additions and document the names we code here in a user-facing way.

We have already a good list of elements to start in WarpX that we can move to ABLASTR and extend.